test/integration/targets/module_utils/runme.sh

```diff
@@ -4,5 +4,36 @@ set -eux
 
 ANSIBLE_ROLES_PATH=../ ansible-playbook module_utils_basic_setcwd.yml -i ../../inventory "$@"
 
+cleanup() {
+    rm -fv /tmp/nulltest
+}
+
+trap cleanup EXIT
+
 ansible-playbook module_utils_test.yml -i ../../inventory -v "$@"
 ANSIBLE_MODULE_UTILS=other_mu_dir ansible-playbook module_utils_envvar.yml -i ../../inventory -v "$@"
+
+# I don't know of an immediate way to get syslog.syslog() to fail on Py3.
+# So limit things to Py2.
+ansible -m setup localhost | grep ansible_python_version | grep '"3' && exit 0
```

Comment:
Put this in an `if` instead of exiting early. Otherwise someone is likely to append tests to the end of this file and not realize they only run on Python 2.

Also, you can consult `${ANSIBLE_TEST_PYTHON_VERSION}` to get the python version being used.